xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Okay. If a friend class is a nogo, solution will be to make ctors public again. Have no better idea. :-(